applications_tests/gls_navier_stokes_3d/CMakeLists.txt

```diff
@@ -1,5 +1,25 @@
 SET(TEST_LIBRARIES lethe)
 SET(TEST_TARGET gls_navier_stokes_3d)
+
+string(TOLOWER ${CMAKE_BUILD_TYPE} CMAKE_LOWER_BUILD_TYPE)
+
 file(COPY cylinder_unstructured.msh DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY poiseuille_restart_files/poiseuille_restart.averagevelocities DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart.flowcontrol DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart.pvdhandler DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart.simulationcontrol DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart.triangulation DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart.triangulation_fixed.data DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart.triangulation.info DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0000.pvtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0005.pvtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0010.pvtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0015.pvtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0000.0000.vtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0005.0000.vtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0010.0000.vtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY poiseuille_restart_files/poiseuille_restart-output.0015.0000.vtu DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${CMAKE_LOWER_BUILD_TYPE})
```

Comment:
This is a lot of binary files to keep track of. Maybe keep only 2 or 3 instead of 4? This is very minor anyway

Reply:
You mean about the pvtu/vtu files? Since average velocities can not be verify with the output text file, they can be useful to observe the flow on Paraview.